code/modules/gear_presets/royal_marines.dm

```diff
@@ -688,7 +688,6 @@
 	new_human.equip_to_slot_or_del(new /obj/item/storage/box/m94, WEAR_IN_BACK)
 
 	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/general/large, WEAR_L_STORE)
-	new_human.equip_to_slot_or_del(new /obj/item/device/defibrillator/compact, WEAR_IN_L_STORE)
```

Comment:
Should probably mention this in the CL?

Reply:
I already discussed it with the this is not intended.